com.microsoft.mrtk.input/Visualizers/Visuals/Models/LICENSE.md.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
+guid: 0fda65c0e56ee1c40a41234fae091059
```

Comment:
Are these the same metas from the original repo? Or have they been regenerated?

Reply:
I don't think we ever got these into a prior version of the MRTK3 repo.

Reply:
Oh whoops, I was asking about the WebXR repo, but then just realized it's not Unity-based so it won't have metas...whoops!